smoketests/__init__.py

```diff
@@ -138,21 +141,6 @@ def run_cmd(*args, capture_stderr=True, check=True, full_output=False, cmd_name=
 def spacetime(*args, **kwargs):
     return run_cmd(SPACETIME_BIN, *args, cmd_name="spacetime", **kwargs)
 
-
-def _check_for_dotnet() -> bool:
```

Comment:
I see we moved this check below where we're processing arguments, that's probably a good place for it 👍